### PR TITLE
Added parameter, 'a' to specify the Server Type either 'ei' or 'microei'

### PR DESCRIPTION
**Pull request by Maninda**
## Purpose
Added Micro EI perf test start parameter 'a' to specify the type of the server. The server type can either be EI or Micro EI as the Micro EI perf test supports Docker deployments for both of them.

## Goals
Perf test both EI and Micro EI Docker images with Micro EI perf tests.

## Approach
Added parameter 'a' to specify the type of the server

### CODE REVIEW

File: distribution/scripts/ei/microei-start.sh [hunk 1]
```diff
@@ -30,10 +31,11 @@ function usage() {
     echo "-v: WSO2 Enterprise Integrator version."
     echo "-m: The heap memory size of Micro Integrator. Default: $default_heap_size."
     echo "-h: Display this help and exit."
+    echo "-a: ei for EI and mei for Micro EI"
     echo ""
 }
 
-while getopts "c:v:m:h" opt; do
+while getopts "c:v:m:ha:" opt; do
```
Comment (chrishantha) [inline distribution/scripts/ei/microei-start.sh]: Let's keep the order here. `h` should be the last one.

File: distribution/scripts/ei/microei-start.sh [hunk 2]
```diff
@@ -94,11 +104,22 @@ capp_dir=$script_dir/capp/
 echo "Starting the docker container:"
 (
     set -x
-    docker run --name=microei -d -p 8280:8290 -p 8243:8253 --add-host=netty:$netty_host --cpus=${cpus} \
-        --volume $(realpath $capp_dir):/home/wso2carbon/wso2ei-${wso2_ei_version}/wso2/micro-integrator/repository/deployment/server/carbonapps \
-        --volume ${HOME}/logs/wso2carbon.log:/home/wso2carbon/wso2ei-${wso2_ei_version}/wso2/micro-integrator/repository/logs/wso2carbon.log \
-        --volume ${HOME}/logs/gc.log:/home/wso2carbon/wso2ei-${wso2_ei_version}/wso2/micro-integrator/repository/logs/gc.log \
-        -e "${JVM_MEM_OPTS}" -e "${JAVA_OPTS}" wso2ei-micro-integrator:${wso2_ei_version}
+    if test "$server_type" = "mei"
```
Comment (chrishantha) [inline distribution/scripts/ei/microei-start.sh]: Do not use `test` command here. Use `[[`. For example:  `if [[ $server_type == "mei" ]]; then`

File: distribution/scripts/ei/microei-start.sh [hunk 3]
```diff
@@ -30,10 +31,11 @@ function usage() {
     echo "-v: WSO2 Enterprise Integrator version."
     echo "-m: The heap memory size of Micro Integrator. Default: $default_heap_size."
     echo "-h: Display this help and exit."
+    echo "-a: ei for EI and mei for Micro EI"
```
Comment (chrishantha) [inline distribution/scripts/ei/microei-start.sh]: Keep this as a flag for now. We only have two options.

**Review by chrishantha — CHANGES_REQUESTED** [review]

File: distribution/scripts/ei/microei-start.sh (hunk 3 above)
Comment (Maninda) [reply distribution/scripts/ei/microei-start.sh]: Corrected.

File: distribution/scripts/ei/microei-start.sh (hunk 2 above)
Comment (Maninda) [reply distribution/scripts/ei/microei-start.sh]: Corrected

File: distribution/scripts/ei/microei-start.sh (hunk 1 above)
Comment (Maninda) [reply distribution/scripts/ei/microei-start.sh]: Corrected

**Review by chrishantha — APPROVED** [review]

**Pull request merged**